docs/basics.rst

```diff
@@ -131,7 +131,7 @@ pixels to 0 and all other pixels to 1 using a simple comparison::
     s2_sceneclassification = (
         connection.load_collection("TERRASCOPE_S2_TOC_V2", bands=["SCENECLASSIFICATION_20M"])
         .filter_temporal(extent=["2016-01-01", "2016-03-10"])
-        .filter_bbox(west=5.1518, east=5.1533,south=51.1819,north=51.1846, crs=4326)
+        .filter_bbox(west=5.1518, east=5.1533,south=51.1819,north=51.1846)
```

Comment:
I would keep these two cases of `crs 4326` because of its documentation value

Reply:
I guess it would make sense to describe somewhere that 4326 is the default and can be omitted - or use another CRS for the example.

Reply:
yeah that line (and other lines)  "need work" anyway: you have to scroll horizontally to see everything on https://open-eo.github.io/openeo-python-client/basics.html#example-applying-a-mask